cmd/app.go

```diff
@@ -658,40 +673,45 @@ func templateInstall(ctx *cli.Context) error {
 		app.TargetNamespace = namespace
 	}
 
+	app.Wait = ctx.Bool("wait")
+	// 300 seconds is the default value that helm already uses
+	if timeout := ctx.Int64("timeout"); timeout != 300 {
+		app.Timeout = timeout
+	}
+
 	madeApp, err := c.ProjectClient.App.Create(app)
 	if err != nil {
 		return err
 	}
 
-	// wait for the app to be installed to print the notes
-	var (
-		timeout   int
-		installed bool
-	)
-	for !installed {
-		if timeout == 60 {
-			return errors.New("timed out waiting for app to install. Run 'rancher apps' to verify status")
-		}
-		timeout++
-		time.Sleep(2 * time.Second)
-		madeApp, err = c.ProjectClient.App.ByID(madeApp.ID)
-		if err != nil {
-			return err
-		}
-		for _, condition := range madeApp.Conditions {
-			condType := strings.ToLower(condition.Type)
-			condStatus := strings.ToLower(condition.Status)
-			if condType == "installed" && condStatus == "true" {
-				installed = true
-				break
-			}
-		}
+	fmt.Printf("run \"app show-notes %s\" to view app notes once app is ready", madeApp.Name)
```

Comment:
Tested and it works, looks good except that I think it should have newline at end. If there is ever another print statement below it then both would be on same line. Current output: 
`run "app show-notes a-gjlhj" to view app notes once app is ready%`

Reply:
fixed